stock_picking_mass_action/models/stock_picking.py

```diff
@@ -10,9 +10,13 @@ class StockPicking(Model):
     _inherit = "stock.picking"
 
     @api.model
-    def check_assign_all(self):
+    def check_assign_all(self, domain=None):
         """Try to assign confirmed pickings"""
-        domain = [("picking_type_code", "=", "outgoing"), ("state", "=", "confirmed")]
+        if not domain:
```

Comment:
@LoisRForgeFlow Should be the domain overridable or completely new one ?

Reply:
Yes. I wanted to say, you don't want to keep the 'outgoing' and 'confirmed' filters and extend the domain ?